packages/client/src/__tests__/integration/happy/signals/__helpers__/client.ts

```diff
@@ -1,20 +1,13 @@
-import { getTestClient } from '../../../../../utils/getTestClient'
 import { EXIT_MESSAGE, READY_MESSAGE } from './constants'
 
-async function main() {
-  const PrismaClient = await getTestClient()
-  const prisma = new PrismaClient()
+const { PrismaClient } = require('../node_modules/@prisma/client')
```

Comment:
That's probably better
```suggestion
import { PrismaClient } from '../node_modules/@prisma/client'
```